aries-site/src/components/home/Hero.js

```diff
@@ -102,7 +102,7 @@ export const Hero = props => {
       >
         <StyledLines
           stroke={normalizeColor('status-unknown', theme, darkMode.value)}
-          fadeStart={sequence(11)}
+          fadesstart={sequence(11)}
```

Comment:
I'm surprised these aren't camelcase anymore. Is that coming from styled components v6?